poms/bom/pom.xml

```diff
@@ -34,6 +34,7 @@
 
     <properties>
         <camel-quarkus.version>2.1.0-SNAPSHOT</camel-quarkus.version><!-- kept in sync with project.version by the release plugin -->
+        <camel-quarkus-community.version>2.1.0</camel-quarkus-community.version>
```

Comment:
Wouldn't it be more practical to move `<camel-quarkus-community.version>2.1.0</camel-quarkus-community.version>` to the top level pom? 

`<camel-quarkus.version>2.1.0-SNAPSHOT</camel-quarkus.version>` is here because it gets auto-updated by the release plugin.